adafruit_rgb_display/ili9341.py

```diff
@@ -54,14 +60,18 @@ class ILI9341(DisplaySPI):
     _ENCODE_POS = ">HH"
     _DECODE_PIXEL = ">BBB"
 
+    #pylint: disable-msg=too-many-arguments
     def __init__(self, spi, dc, cs, rst=None, width=240, height=320,
                  baudrate=16000000, polarity=0, phase=0):
         super().__init__(spi, dc, cs, rst=rst, width=width, height=height,
                          baudrate=baudrate, polarity=polarity, phase=phase)
         self._scroll = 0
+    #pylint: enable-msg=too-many-arguments
 
-    def scroll(self, dy=None):
-        if dy is None:
+    def scroll(self, delta_y=None):
```

Comment:
This is "dy" not "delta y"  or "δy". It's a common notation in math.

Reply:
reverted to dy